<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblActionDesc">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>{Copy} from:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblFileSrc">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans Mono</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>  sourceFile</string>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_8">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>To:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblFileTgt">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans Mono</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>  targetFile</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblMsg">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>The following {file_directory} already exists</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: palette(light);
padding: 6;</string>
        </property>
        <property name="text">
         <string>Source</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblFileSrcImg">
        <property name="text">
         <string>-preview source-</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="lblFileTgtNfo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans Mono</family>
         </font>
        </property>
        <property name="text">
         <string>&lt;b&gt;Date:&lt;/b&gt; yyyy-mm-dd hh:mi:ss&lt;br&gt;&lt;b&gt;Size:&lt;/b&gt; 0B&lt;br&gt;&lt;b&gt;Image size:&lt;/b&gt; 0x0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: palette(light);
padding: 6;</string>
        </property>
        <property name="text">
         <string>Destination</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblFileSrcNfo">
        <property name="font">
         <font>
          <family>DejaVu Sans Mono</family>
         </font>
        </property>
        <property name="text">
         <string>&lt;b&gt;Date:&lt;/b&gt; yyyy-mm-dd hh:mi:ss&lt;br&gt;&lt;b&gt;Size:&lt;/b&gt; 0B&lt;br&gt;&lt;b&gt;Image size:&lt;/b&gt; 0x0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lblFileTgtImg">
        <property name="text">
         <string>-preview target-</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Rename as:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbxNewName">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans Mono</family>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Define default name applied to new created document&lt;/p&gt;&lt;hr/&gt;&lt;p&gt;&lt;span style=&quot; font-family:'monospace'; font-size:8pt; font-weight:600;&quot;&gt;{file:baseName} &lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;The file base name without extension&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'monospace'; font-size:8pt; font-weight:600;&quot;&gt;{file:ext}      &lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;The file extension&lt;/span&gt;&lt;/p&gt;&lt;hr/&gt;&lt;p&gt;&lt;span style=&quot; font-family:'monospace'; font-size:8pt; font-weight:600;&quot;&gt;{date}          &lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;The current system date (&lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-style:italic;&quot;&gt;yyyy-mm-dd&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;)&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'monospace'; font-size:8pt; font-weight:600;&quot;&gt;{date:yyyy}     &lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;The current system &lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-style:italic;&quot;&gt;year&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'monospace'; font-size:8pt; font-weight:600;&quot;&gt;{date:mm}       &lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;The current system &lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-style:italic;&quot;&gt;month&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'monospace'; font-size:8pt; font-weight:600;&quot;&gt;{date:dd}       &lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;The current system &lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-style:italic;&quot;&gt;day&lt;/span&gt;&lt;/p&gt;&lt;hr/&gt;&lt;p&gt;&lt;span style=&quot; font-family:'monospace'; font-size:8pt; font-weight:600;&quot;&gt;{time}          &lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;The current system &lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-style:italic;&quot;&gt;time &lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;(&lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-style:italic;&quot;&gt;hh:mm:ss&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;)&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'monospace'; font-size:8pt; font-weight:600;&quot;&gt;{time:hh}       &lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;The current system &lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-style:italic;&quot;&gt;hour (00-24)&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'monospace'; font-size:8pt; font-weight:600;&quot;&gt;{time:mm}       &lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;The current system &lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-style:italic;&quot;&gt;minutes&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'monospace'; font-size:8pt; font-weight:600;&quot;&gt;{time:ss}       &lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;The current system &lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-style:italic;&quot;&gt;seconds&lt;/span&gt;&lt;/p&gt;&lt;hr/&gt;&lt;p&gt;&lt;span style=&quot; font-family:'monospace'; font-size:8pt; font-weight:600;&quot;&gt;{size}          &lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;The current image size (&lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-style:italic;&quot;&gt;widthxheight&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;)&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'monospace'; font-size:8pt; font-weight:600;&quot;&gt;{size:width}    &lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;The current image &lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-style:italic;&quot;&gt;width&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'monospace'; font-size:8pt; font-weight:600;&quot;&gt;{size:height}   &lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;The current image &lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-style:italic;&quot;&gt;height&lt;/span&gt;&lt;/p&gt;&lt;hr/&gt;&lt;p&gt;&lt;span style=&quot; font-family:'monospace'; font-size:8pt; font-weight:600;&quot;&gt;{counter}       &lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;A counter to file name&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'monospace'; font-size:8pt; font-weight:600;&quot;&gt;{counter:####}  &lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;A counter to file name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="cbApplyToAll">
        <property name="text">
         <string>Apply to all</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btActionRename">
        <property name="text">
         <string>Rename</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btActionSkip">
        <property name="text">
         <string>Skip</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btActionOverwrite">
        <property name="text">
         <string>Overwrite</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btActionAbort">
        <property name="text">
         <string>Abort</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
